src/app/docs/dropdown/dropdown-docs.component.html

```diff
@@ -0,0 +1,330 @@
+          caption="use icon-only dropdowns when there is not space to include a label."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p>
+          Don't use dropdowns as form inputs. Use HTML <sky-code>&lt;select&gt;</sky-code> fields instead.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/dropdown/usage-icon.png"
+          caption="use dropdowns as form inputs."
+          captionType="danger"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Anatomy"
+    >
+      <sky-docs-anatomy
+        thumbnailPath="~/assets/img/guidelines/dropdown/anatomy.png"
+      >
+        <sky-docs-anatomy-item>Dropdown button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Dropdown menu</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item>Action</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Options"
+    >
+      <sky-docs-design-guideline
+        heading="Button types"
+      >
+      <table class="sky-docs-table">
+          <tr>
+            <td class="sky-docs-table-cell">Default</td>
+            <td class="sky-docs-table-cell">
+                <sky-docs-design-guideline-thumbnail
+                  imageSource="~/assets/img/guidelines/dropdown/option-type-default.png"
+                >
+                </sky-docs-design-guideline-thumbnail>
+              </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">Context menu</td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/dropdown/option-type-context.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">Icon-only</td>
+            <td class="sky-docs-table-cell">
+                <sky-docs-design-guideline-thumbnail
+                  imageSource="~/assets/img/guidelines/dropdown/option-type-icon.png"
+                >
+                </sky-docs-design-guideline-thumbnail>
+              </td>
+          </tr>
+        </table>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline
+        heading="Button styles"
+      >
+        <p>Dropdown buttons can use the default or primary styles inherited from SKY UX buttons. For details about when to use the primary style, see the <a routerLink="/design/guidelines/page-layouts" fragment="primary-action-guidelines">primary action guidelines</a>.</p>
```

Comment:
This will need to be a hard URL (`href`) pointing to the page on the main docs site.